guides/source/active_record_basics.md

```diff
@@ -249,37 +249,37 @@ purposes.
 
 Active Record models are placed under the `app/models` directory by default. But
 you may want to organize your models by placing similar models under their own
-folder and namespace. For example, `order.rb` and `review.rb` under
-`app/models/product` with `Product::Order` and `Product::Review` class names,
+folder and namespace. For example, `book.rb` and `magazine.rb` under
+`app/models/product` with `Product::Book` and `Product::Magazine` class names,
```

Comment:
Oh, I think I misunderstood what you meant when [you commented on the other PR](https://github.com/rails/rails/pull/51226#discussion_r1527578109)... I thought the intention was to change `Product` => `Book`, in other words, use `Book::Order` and `Book::Review`.

I'm not sure about the example specializing on types of products 🤔

Reply:
Done. `Book::Order` and `Book::Review` make sense to me too.